04_environment/design.tex

```diff
@@ -0,0 +1,289 @@
+
+\begin{equation}
+\text{Increments of catching $n$ animals}=
+\left( \begin{array}{ll@{}}
+n=0 \longrightarrow \Delta^0 = 1 \\
+n=1 \longrightarrow \Delta^1 = 0.8^1 \\
+n=2 \longrightarrow \Delta^2 = 0.8^2 = 0.64 \\
+n=2 \longrightarrow \Delta^3 = 0.8^3 = 0.512 \\
+n=4 \longrightarrow \Delta^4 = 0.8^4 = 0.4096\\
+\end{array} \right) 
+\label{eq:Cumulative Cost 1}
+\end{equation}
+
+\begin{equation}
+\text{Utility Tier Cost}=\ \sum_{n=0}^{n} \Delta^{n} = \Delta^{0} + \Delta^{1} + \Delta^{2} + \Delta^{3} + \Delta^{4} + ....
+\label{eq:Cumulative Cost 2}
+\end{equation}
+
+Equation~\eqref{eq:Cumulative Cost 1} and Equation~\eqref{eq:Cumulative Cost 2} show the formulation of the decay function for tier system. The tier system is shown in the Figure~\ref{fig:Foraging Tier System} where $n$ stands for the tiers, with decay costs ($\Delta$) of $0.8$ for the deer hunt and $0.6$ for fishing.
+
+\begin{figure}[!htb]
+    \centering
+    \includegraphics[width=0.9\textwidth]{04_environment/images/Foraging Tier System.PNG}
+    \caption{Foraging Tier System}
+    \label{fig:Foraging Tier System}
+\end{figure}
+
+The amount of resources inputted is scaled using a variable and the tiers are then calculated with them, the higher the tiers the lower the cost to catch the next animal. However, the total cost increases up to a daily catch limit. This means that if teams collaborate, they can invest less as individuals to get a collectively greater return through deer hunting. If they invest too much, then they will over spend as they reach the daily catch limit. As a result, it is more beneficial for some teams to go fishing because it has a higher daily catch limit, although it comes with lower returns. In other words, there is a need for some self-sacrifice to reach the maximum welfare (see Definition~\ref{def:Welfare}). Moreover, fishing is a similar foraging method with the key difference being that it utilised a normal distribution and the start of the tiers.
+
+\subsection{Example Distribution}
+\subsubsection{Example of the Distribution for Foraging Return (Deer Hunting - Payoff Dominant)}
+
+\begin{figure}[!htb]
+    \centering
+    \includegraphics[width=1\textwidth]{04_environment/images/Distribution of Foraging returns Deer Hunting.PNG}
+    \caption{Deer hunting payoff dominant Distribution for foraging return}
+    \label{fig:Distribution of Foraging returns Deer Hunting}
+\end{figure}
+
+To increase the risk of deer hunting, a Bernoulli Random Variable ($D$) was used to prevent guaranteed resource returns after a hunt. While the Exponential Decay ($W$) is to disincentive too much resources being placed in hunting and hoping for a high return. However, this all comes with the benefit of significantly higher ($2\times$) returns than fishing.
```

Comment:
The second sentence here is a bit confusing. Suggested rewrite:

The Exponential Decay ($W$) disincentives too many resources being allocated to hunting in the hope of a higher return.